content/pages/conference/report.rst

```diff
@@ -0,0 +1,213 @@
+        }   
+    </style>
+    <div class="grid-container">
+        <div class="grid-item">
+            <a data-flickr-embed="true" data-header="true" data-context="true" href="https://www.flickr.com/photos/sciwork/53399144411/in/album-72177720313419664/" title="_DSF3620"><img src="https://live.staticflickr.com/65535/53399144411_55f1043df0_z.jpg" width="640" height="427" alt="_DSF3620"/></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>
+            <span>Check-in</span>
+        </div>
+        <div class="grid-item">
+            <a data-flickr-embed="true" data-header="true" data-context="true" href="https://www.flickr.com/photos/sciwork/53399578990/in/album-72177720313419664/" title="_DSF3668"><img src="https://live.staticflickr.com/65535/53399578990_093bd486a0_z.jpg" width="640" height="427" alt="_DSF3668"/></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>
+            <span>Opening</span>
+        </div>
+        <div class="grid-item">
+            <a data-flickr-embed="true" data-header="true" data-context="true" href="https://www.flickr.com/photos/sciwork/53398239117/in/album-72177720313393655/" title="IMG_0896"><img src="https://live.staticflickr.com/65535/53398239117_08e127d7ef_z.jpg" width="640" height="427" alt="IMG_0896"/></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>
+            <span>Group Photo</span>
+        </div>
+        <div class="grid-item">
+            <a data-flickr-embed="true" data-header="true" data-context="true" href="https://www.flickr.com/photos/sciwork/53399359903/in/album-72177720313393655/" title="IMG_0879"><img src="https://live.staticflickr.com/65535/53399359903_aa0aa996cd_z.jpg" width="640" height="427" alt="IMG_0879"/></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>
+            <span><a href="https://humbledata.org">Humble Data</a> Workshop</span>
+        </div>
+    </div>
+
+Humble Data
+-----------------------------
+
+We thank all the participants, including sponsors, volunteers, and contributors.
+We especially appreciate NYCU CS department for their help for the venue.
+The conference also went with the `Humble Data <https://humbledata.org>`__ workshop.
+The full list of the program is available at https://conf.sciwork.dev/program.
+
+
+Scientific Programming
+--------------------------------
+
+The community of sciwork focuses on scientific programming, which is a less visible area. 
+Electronic computers were invented for engineering purposes,
+and programming for science and engineering is not only important for solving the problems,
+but also leads to advances to computing itself.
+Although the rewards are huge, it does take a lot of effort to get into the business of scientific software development.
+
+
+
+.. figure:: https://github.com/sciwork/swportal/assets/18013815/c55d5183-0989-4521-a013-6115c3278800
```

Comment:
Assets should be added to the repository along with the text.

Reply:
fixed.